### PR TITLE
Updated deploy script

### PR DESCRIPTION
**Pull request by markyd-XXI**

### CODE REVIEW

File: .github/workflows/vue_deploy.yml [hunk 1]
```diff
@@ -5,22 +5,22 @@ on:
...
   aws-deploy:
     runs-on: ubuntu-latest
     #For google auth
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         # may need to change to version 3
         with:
           ref: dev
```
Comment (n-grounds) [inline .github/workflows/vue_deploy.yml:19]: this line here says that you're going to checkout the `dev` branch (on each pull request merged to `main`) and do a build and deployment

File: .github/workflows/vue_deploy.yml [hunk 2]
```diff
@@ -5,22 +5,22 @@ on:
...
 
 jobs:
   aws-deploy:
     runs-on: ubuntu-latest
     #For google auth
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         # may need to change to version 3
```
Comment (n-grounds) [inline .github/workflows/vue_deploy.yml:17]: no need to keep this comment and the one on line 21 anymore

Comment (markyd-XXI) [reply .github/workflows/vue_deploy.yml]: Updated. Thanks for the feedback.

File: .github/workflows/vue_deploy.yml (hunk 1 above)
Comment (markyd-XXI) [reply .github/workflows/vue_deploy.yml]: Thanks for the catch! I though I did that.

**Review by n-grounds — APPROVED** [review]

**Comment by n-grounds** [comment]
Someone forgot to rebase and squash before merging:
image

**Pull request merged**